### PR TITLE
feat: Added Input component

### PR DESCRIPTION
**Pull request by krishanu2911**
Added Input component with variations of default, error, and textbox (text area)
Screenshot (417)

### CODE REVIEW

**Review by simrk — COMMENTED** [review]
Lgtm.

File: CSS/components/input.css [hunk 1]
```diff
@@ -0,0 +1,28 @@
+.input-form{
+    display: flex;
+    width: 20rem;
+    height: max-content;
+    flex-direction: column;
+}
+.input-lable{
+    display: block;
+}
```
Comment (shuntzhere) [inline CSS/components/input.css]: You might want to check this classname.

**Review by shuntzhere — COMMENTED** [review]
Gave a small suggestion to change the class name.

File: CSS/components/input.css (hunk 1 above)
Comment (krishanu2911) [reply CSS/components/input.css]: oh ya thank you so much Masoom It's a  typo!!!!

Comment (krishanu2911) [reply CSS/components/input.css]: corrected it

**Review by shuntzhere — APPROVED** [review]
Looks good now.

**Pull request merged**